newIDE/app/src/ResourcesList/ResourceSelector.js

```diff
@@ -168,15 +168,16 @@ export default class ResourceSelector extends React.Component<Props, State> {
     const resourcesManager = project.getResourcesManager();
     const initialResource = resourcesManager.getResource(resourceName);
 
-    let initialResourceMetadata = {};
+    let metadata = {};
```

Comment:
Any reason why you can't keep this named `initialResourceMetadata`? Was it less clear?

Reply:
haha I made it even less clear xD no reason, just wanted a generic name for it with less meaning to communicate 'initialResource'

Reply:
Ok I am going to update the name everywhere to make it more explicit sorry:)